dotCMS/src/integration-test/java/com/dotmarketing/util/PageModeTest.java

```diff
@@ -111,6 +114,59 @@ public void test_page_mode_does_not_create_session() {
 
     }
 
+    /**
+     * Back end users can have their page mode set to EDIT MODE
+     */
+    @Test
+    public void test_back_end_user_can_be_set_to_LIVE() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6106392)